src/cmp_aus.js

```diff
@@ -130,6 +135,7 @@ const loadPage = async (page, url) => {
 	// Clear cookies before starting testing, to ensure the CMP is displayed.
 	const client = await page.target().createCDPSession();
 	await clearCookies(client);
+	await clearLocalStorage(page);
```

Comment:
Doesn't have to be now but we could call these two functions in a single `teardown()` function that is called by each test.

Reply:
If it works 🤞

Reply:
Thank you @kenoir!

So SP only saves to localstorage in a TCFv2 region, and not for the others?

Reply:
@arelra this may mean we can't go with your `teardown` suggestion since we call localstorage later in the canary run.

Reply:
I believe it also uses LocalStorage in the CCPA jurisdiction, but the thing to note is that you can't clear LocalStorage before you've loaded a page (whereas you can with cookies). You could probably implement a generic resetting function that looks like:

```ts
const resetPage = async (client: CDPSession, page: Page) => {
  await loadPage(page, url);
  await clearCookies(client);
  await clearLocalStorage(page);
  await loadPage(page, url);
}
```